__tests__/app.test.js

```diff
@@ -83,14 +82,85 @@ describe("4. GET /api/reviews/:review_id", () => {
       .then((response) => {
         expect(response.body.msg).toBe("Invalid review ID provided");
       });
-  })
-test("When user submits a valid request but there is not a matching review ID, function to notify user",()=>{
-  
-  return request(app)
-  .get("/api/reviews/100")
-  .expect(200)
-  .then((response) => {
-    expect(response.body.msg).toBe("No matching review id");
+  });
+  test("When user submits a valid request but there is not a matching review ID, function to notify user", () => {
+    return request(app)
+      .get("/api/reviews/100")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.msg).toBe("No matching review id");
+      });
+  });
+});
+
+describe("6. GET /api/reviews/:review_id/comments", () => {
+  test("Function gives requested information when given the correct ID", () => {
+    return request(app)
```

Comment:
This test is too brittle, as you have hard-coded this. It would be better to use the test data to check that the properties match what you expect. 👍